src/psyclone/dynamo0p3.py

```diff
@@ -3614,12 +3967,18 @@ def basis(self, function_space):
                 "expecting one of {0} but found "
                 "'{1}'".format(VALID_FUNCTION_SPACES,
                                function_space.orig_name))
+        if self._kern.eval_shape in VALID_QUADRATURE_SHAPES:
```

Comment:
document args in parent basis method docstring

Reply:
Done.